### PR TITLE
Refactor external add-ons section

### PR DESCRIPTION
**Pull request by thesuperzapper**
This PR refactors and cleans up the "External Add-Ons" section.

The key changes this PR mages:

1. Gives each project its own section, with __3 links__ on the sidebar:
     - __Introduction__ - a brief overview of `What is XXXX?` and `"How to use XXXX with Kubeflow?"`
     - __Website__ - a link to the project's docs website
     - __GitHub Repo__ - a link to the open-source projects GitHub repo
2. Creates the following sections under External Add-Ons:
     - KServe
     - Feast
     - Elyra
3. Projects which were removed:
     - [BentoML Yati](https://github.com/bentoml/Yatai) (not open source - [Elastic v2 license](https://github.com/bentoml/Yatai/blob/main/LICENSE.md), and is also abandoned)
     - [Seldon Core](https://github.com/SeldonIO/seldon-core) (not open source - [Business Source License 1.1](https://github.com/SeldonIO/seldon-core/blob/master/LICENSE))
     - [MLRun](https://github.com/mlrun/mlrun) (seemingly a competing project to Kubeflow, unclear why it was ever included)
         - If the KSC decides they still want to elevate a competing project, I have a single commit which can be reverted to add a section for MLRun.
     - All other specific serving runtimes (as they are included in KServe)

In the "Introduction" pages, I have largley left the wording unchanged, and mostly tried to structure everything under the two core headings: `"What is XXXX?"`, and `"How do I use XXXX with Kubeflow?"`

# Screenshots

## Section Overview

External-Add-Ons-Kubeflow

## Feast Section

Feast-Kubeflow

### CODE REVIEW

**Comment by thesuperzapper** [comment]
@andreyvelich @franciscojavierarceo @jbottum this is the PR to clean up the External Add-Ons section, I would appreciate your review.

See the preview site here:
- https://deploy-preview-3836--competent-brattain-de2d6d.netlify.app/docs/external-add-ons/

File: content/en/docs/external-add-ons/feast/introduction.md
Comment (franciscojavierarceo) [inline content/en/docs/external-add-ons/feast/introduction.md:13]:
```suggestion
- __Load streaming, batch, and request-time data__: Feast is built to be able to ingest data from a variety of bounded or unbounded sources. 
```

File: content/en/docs/external-add-ons/feast/introduction.md
Comment (franciscojavierarceo) [inline content/en/docs/external-add-ons/feast/introduction.md:14]:
```suggestion
  Feast allows users to ingest data from api-calls, streams, object stores, databases, or notebooks. 
```

File: content/en/docs/external-add-ons/feast/introduction.md
Comment (franciscojavierarceo) [inline content/en/docs/external-add-ons/feast/introduction.md:15]:
```suggestion
  Data that is ingested into Feast can be persisted in both online store and historical stores, which in turn is used for the creation of training datasets and serving features to online systems.
```

File: content/en/docs/external-add-ons/feast/introduction.md
Comment (franciscojavierarceo) [inline content/en/docs/external-add-ons/feast/introduction.md:31]:
```suggestion
- __Feature sharing and reuse__: Feast provides a discovery and metadata API that allows teams to track, share, and reuse features across projects. Feast also provides a light-weight web UI to expose this metadata.
```

File: content/en/docs/external-add-ons/feast/introduction.md
Comment (franciscojavierarceo) [inline content/en/docs/external-add-ons/feast/introduction.md:47]:
```suggestion
- __Serving features at scale__: Models need data that can come from a variety of sources, including api-calls, event streams, data lakes, warehouses, or notebooks. 
```

File: content/en/docs/external-add-ons/feature-store/OWNERS
Comment (franciscojavierarceo) [inline content/en/docs/external-add-ons/feature-store/OWNERS:3]: Can we keep this one?

**Comment by franciscojavierarceo** [comment]
> @andreyvelich @franciscojavierarceo @jbottum this is the PR to clean up the External Add-Ons section, I would appreciate your review.
> 
> See the preview site here:
> 
> * https://deploy-preview-3836--competent-brattain-de2d6d.netlify.app/docs/external-add-ons/

Thank you for this! I left some small nits.

**Comment by rimolive** [comment]
Changes look great to me, but my initial concern is still valid. There's missing instructions on how to deploy and integrate Feast with Kubeflow. @franciscojavierarceo any change to provide these instructions in Feast docs while we work to bring the manifests back in Kubeflow?

**Comment by franciscojavierarceo** [comment]
> Changes look great to me, but my initial concern is still valid. There's missing instructions on how to deploy and integrate Feast with Kubeflow. @franciscojavierarceo any change to provide these instructions in Feast docs while we work to bring the manifests back in Kubeflow?

I'm going to make a follow up PR with how to do this after I add the manifests back in but we shouldn't have to block on that imo.

**Comment by franciscojavierarceo** [comment]
But @dmartinol @jeremyary we should discuss that

**Comment by ederign** [comment]
I really like the structural changes @thesuperzapper !

/lgtm

File: content/en/docs/external-add-ons/feast/introduction.md
Comment (thesuperzapper) [reply content/en/docs/external-add-ons/feast/introduction.md]: Done in https://github.com/kubeflow/website/pull/3836/commits/a5e72b551cea20a364a90aed4f6ef4f558a2582e

File: content/en/docs/external-add-ons/feast/introduction.md
Comment (thesuperzapper) [reply content/en/docs/external-add-ons/feast/introduction.md]: Done in https://github.com/kubeflow/website/pull/3836/commits/a5e72b551cea20a364a90aed4f6ef4f558a2582e

File: content/en/docs/external-add-ons/feast/introduction.md
Comment (thesuperzapper) [reply content/en/docs/external-add-ons/feast/introduction.md]: Done in https://github.com/kubeflow/website/pull/3836/commits/a5e72b551cea20a364a90aed4f6ef4f558a2582e

File: content/en/docs/external-add-ons/feast/introduction.md
Comment (thesuperzapper) [reply content/en/docs/external-add-ons/feast/introduction.md]: Done in https://github.com/kubeflow/website/pull/3836/commits/a5e72b551cea20a364a90aed4f6ef4f558a2582e

File: content/en/docs/external-add-ons/feature-store/OWNERS
Comment (thesuperzapper) [reply content/en/docs/external-add-ons/feature-store/OWNERS:3]: The purpose of these sections is not to provide up to date information, it is to provide a brief "evergreen" explanation of what the add-on is, and how it relates to Kubeflow.

I would prefer if the root approvers of the Kubeflow website are aware and approve each proposed change, to avoid messaging conflicts.

Even without being an approver, you or others can raise PRs to propose updates.

**Comment by thesuperzapper** [comment]
@franciscojavierarceo I have applied your updates, and added the diagram, please see the new preview site:

- https://deploy-preview-3836--competent-brattain-de2d6d.netlify.app/docs/external-add-ons/

**Review by franciscojavierarceo — APPROVED** [review]
/lgtm

Thanks for this!!

**Comment by thesuperzapper** [comment]
@terrytangyuan we need a root approver because I had to fix files outside the KFP directory to avoid broken links.

I'm pretty sure this is ready to merge now.

**Review by terrytangyuan — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**
